### DISCUSSION TITLE
[Directory Hotlist]: WFX view

### DISCUSSION THREAD

**Discussion by PetbkA** [doublecmd/doublecmd#1906] [Suggestions]
It would be nice to have Directory Hotlist as WFX plugin. Possible benefits:
1. Quickly find hotdirs via filter and Flat View
2. Copying submenus (related to #1193);
3. Rename hotdirs and submenus one by one and via Multi-Rename Tool;
4. Changing paths one by one (via *Properties* / `Alt+Enter`) and via replace in files in search menu

**Comment by j2969719** [comment]
a grain of salt: the hotlist is stored in doublecmd.xml wich is loaded on dc startup and is saved on exit by default, all changes to the config by a third-party plugin are kinda useless. wfx does not know how to search, show flatview (and multi-rename with wdx plugins if the selected files are not links to local files). not to mention the limitations of the api such as the length of filenames, strange work with symlinks, the need to implement copying in the plugin even for local files, lack of support for moving files from wfx, etc.
